R/opq.R

```diff
@@ -18,6 +19,9 @@
 #' \dontrun{
 #' overpass_query(q) -> reading_noms
 #' sp::plot(reading_noms$osm_nodes)
+#' p <- overpass_query(opq("Leeds") %>%
+#'   add_feature("leisure", "park"))
+#' sp::plot(p$osm_polygons[1,])
```

Comment:
```
require (sp)
sldf <- new ("SpatialLinesDataFrame")
lst <- list (lines1=sldf, lines2=sldf)
lst [[1]]
# gives appropriate output
lst
# $lines1
# Error in ...
```
That's why the `print` method doesn't work properly. You `sp::plot` problem will be for similar reasons